srcdoc/configure/h2olog.mt

```diff
@@ -0,0 +1,85 @@
+? my $ctx = $main::context;
+? $_mt->wrapper_file("wrapper.mt", "Configure", "Using h2olog for Tracing")->(sub {
+
+<p>h2olog is a <a href="https://www.kernel.org/doc/html/latest/bpf/index.html">BPF</a> (<a href="https://www.kernel.org/doc/Documentation/networking/filter.txt">kernel doc</a>) backed HTTP request tracing tool for the <a href="https://github.com/h2o/h2o">H2O</a> server.
+h2olog can also be used to log <a href="https://en.wikipedia.org/wiki/QUIC">QUIC</a> events for <a href="https://en.wikipedia.org/wiki/Transport_layer">transport layer</a> observation.
+See <a href="#tracing-quic-events">Tracing QUIC events</a> for how.</p>
+
+<h2 id="installing-from-source">Installing from Source</h2>
+
+<p>See <a href="#requirements">requirements</a> for build prerequisites. If dependencies are satisfied, <code>-DWITH_H2OLOG=on</code> will be set and the <code>h2olog</code> target will be configured.</p>
+<p>If you have <code>BCC</code> installed to a non-standard path, use <code>pkg-config</code> for <code>cmake</code>.
+
+<?= $ctx->{code}->(<<'EOT')
+$ PKG_CONFIG_PATH=/path/to/bcc/lib/pkgconfig cmake [options]
+EOT
+?>
+
+<h2 id="requirements">Requirements</h2>
+
+<h3>For building h2olog</h3>
+<ul>
+<li>C++11 compiler</li>
+<li>CMake (&gt;= 3.8) for generating the build files</li>
+<li>pkg-config for detecting dependencies</li>
+<li>Python 3 for the code generator</li>
+<li><a href="https://iovisor.github.io/bcc/">BCC (a.k.a. bpfcc)</a>(&gt;= 0.11.0) <a href="https://github.com/iovisor/bcc/blob/master/INSTALL.md">installed</a> on your system</li>
+</ul>
+<p>For Ubuntu 20.04 or later, you can install dependencies with:</p>
+<?= $ctx->{code}->(<<'EOT')
+$ sudo apt install clang cmake python3 libbpfcc-dev linux-headers-$(uname -r)
+EOT
+?>
+<h3>For running h2olog</h3>
+<ul>
+<li>Root privilege to execute the program</li>
```

Comment:
I think we might want to say "h2olog" rather than "the program" to avoid confusion, as we would be running multiple programs (e.g. h2o).